automating_bulk_processor.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+python -m toolbox.bulk_processing.refusal_processor
+ret_value=$?
+if [ $ret_value -ne 0 ];
+then
+  echo "refusal_processor has failed"
+fi
+python -m toolbox.bulk_processing.new_address_processor
+ret_value=$?
+if [ $ret_value -ne 0 ];
+then
+  echo "new_address_processor has failed"
+fi
+python -m toolbox.bulk_processing.invalid_address_processor
+ret_value=$?
+if [ $ret_value -ne 0 ];
+then
+  echo "invalid_address_processor has failed"
+fi
+python -m toolbox.bulk_processing.deactivate_uac_processor
+ret_value=$?
+if [ $ret_value -ne 0 ];
+then
+  echo "deactivate_uac_processor has failed"
+fi
+python -m toolbox.bulk_processing.address_update_processor
+ret_value=$?
+if [ $ret_value -ne 0 ];
+then
+  echo "address_update_processor has failed"
+fi
+python -m toolbox.bulk_processing.uninvalidate_address_processor
+ret_value=$?
+if [ $ret_value -ne 0 ];then
+    echo "uninvalidate_address_processor has failed"
+fi
```

Comment:
the script can/should return a non-zero exit code so the k8s job indicates an error

Reply:
I've just updated the script now to error on failure so give it a go and see how it runs. I had to change the k8s cronjob yaml to have a backoff of 0 to allow it to fail correctly